### DISCUSSION TITLE
Issue - can't deploy edge function.

### DISCUSSION THREAD

**Discussion by laurencebedford** [supabase/supabase#20076] [Questions]
Hey supabase community I recently deployed an edge function and it worked beautifully. However, when I came to deploy it again - to push some updated code it wouldn't work. 

I then went to the dashboard and manually deleted the edge function and tried to deploy it again via CLI however this doesn't work and I get the following error:

`Failed to create a new Function on the Supabase project: {"message":"Cannot POST /v1/projects//functions?entrypoint_path=file%3A%2F%2F%2Fhome%2Fdeno%2Ffunctions%2Fmark-quiz%2Findex.ts&import_map_path=file%3A%2F%2F%2Fhome%2Fdeno%2Fimport_map.json&name=mark-quiz&slug=mark-quiz&verify_jwt=true"}`

I have then since followed the rouble shooting guide:
Unable to deploy Edge Function[#](https://supabase.com/docs/guides/functions/debugging#unable-to-deploy-edge-function)
Make sure you're on the latest version of the [Supabase CLI](https://supabase.com/docs/guides/cli#updates).
Run the deploy command with the --debug flag.
Run the deploy command with the --legacy-bundle flag.
If the output from the commands above does not help you to resolve the issue, open a support ticket via the Supabase Dashboard (by clicking the "Help" button at the top right) and include all output from the commands mentioned above.

Does anyone know why its failing?

**Comment by abid365** [comment]
I'm just facing the same issue , mine does not deployed nor the supabse cli works, when I try to deploy a fn it show a pop that this app is not compatible for PC. I deployed the edge fn 46 times before but never encountered this.
